### PR TITLE
fix(generators/readme): Escape the "|" character while join enum values

### PR DESCRIPTION
**Pull request by mohib0306**
#### Summary

-`generators/readme` => escape the pipe character properly while joining `enum` values in the readme generation process

### CODE REVIEW

**Review by emmenko — COMMENTED** [review]
You have to regenerate the readmes

File: generators/readme/src/index.ts [hunk 1]
```diff
@@ -314,7 +314,7 @@ const parsePropTypesToMarkdown = (componentPath: string) => {
             inlineCode(
               (propInfo.type.value as { value: string }[])
                 .map((enumValue) => enumValue.value)
-                .join(' | ')
+                .join('\\|')
```
Comment (emmenko) [inline generators/readme/src/index.ts:317]:
```suggestion
                .join(' \\| ')
```

**Review by emmenko — COMMENTED** [review]
Thanks for spotting that!

**Review by tdeekens — APPROVED** [review]

**Comment by tdeekens** [comment]
Apart from @emmenko suggestion this looks good.

**Comment by mohib0306** [comment]
> You have to regenerate the readmes

Yes I am going through each package to regenerate the readmes

**Comment by emmenko** [comment]
Thanks.

PS: In the other PR #1509 I added a general command to generate the readmes in each package.

**Comment by emmenko** [comment]
@mohib0306 I merged the other PR. You can rebase this branch and use the `yarn generate-readmes` command.

**Review by emmenko — APPROVED** [review]
💯

**Comment by mohib0306** [comment]
Do we need changeset?

**Comment by emmenko** [comment]
> Do we need changeset?

Not strictly. I think we can skip it in this case.

**Pull request merged**